src/components/SectionSwap.tsx

```diff
@@ -18,19 +18,20 @@ import {
   toPurseAtom,
 } from 'store/swap';
 
-export const SectionSwapType = {
-  FROM: 'FROM',
-  TO: 'TO',
-} as const;
+export enum SectionSwapType {
+  FROM = 'FROM',
+  TO = 'TO',
+}
 
-const SectionSwap = ({ type }: { type: string }) => {
+const SectionSwap = ({ type }: { type: SectionSwapType }) => {
   const { displayBrandIcon, displayBrandPetname } =
     useAtomValue(displayFunctionsAtom);
   const fromPurse = useAtomValue(fromPurseAtom);
   const toPurse = useAtomValue(toPurseAtom);
   const [toAmount, setToAmount] = useAtom(toAmountAtom);
   const [fromAmount, setFromAmount] = useAtom(fromAmountAtom);
   const [open, setOpen] = useState(false);
+  const multiBrandsEnabled = useAtomValue(previewEnabledAtom);
```

Comment:
at first I wasn't sure about renaming this value but it makes sense to infer as it pertains to this scope